bender-sim.mk

```diff
@@ -8,7 +8,9 @@
 # bender targets
 sim_targs += -t sim
 sim_targs += -t test
+sim_targs += -t snitch_cluster
 sim_targs += -t simulation
+sim_targs += -t carfield_l2dual_secure_periph
```

Comment:
Is there a reason for changing configuration?

Reply:
in this config spatz is disabled, otherwise it would not compile as still compliant with old release of iDMA. we can discuss which config to use as default.

Reply:
I would suggest putting it [here](https://github.com/pulp-platform/astral/blob/astral/bender-common.mk#L9) so that the updated implementation is visible in all the steps!

Reply:
fixed